### ISSUE TITLE
Incorrect eslint error for navigator.mediaDevices

### ISSUE THREAD

**Issue by danielsht86** [amilajack/eslint-plugin-compat#405]
We currently have the following line of code:
```
        const devices = await navigator.mediaDevices.enumerateDevices();
```

which logs this eslint error.

`64:31  error  navigator.mediaDevices() is not supported in Safari 13  compat/compat`

However, according to caniuse, this API is available in Safari 13
https://caniuse.com/?search=mediadevices

Thanks!
